Modix.Data/Models/Core/ConfigurationActionSummary.cs

```diff
@@ -13,6 +13,11 @@ public class ConfigurationActionSummary
         /// </summary>
         public long Id { get; set; }
 
+        /// <summary>
+        /// See <see cref="ConfigurationActionEntity.GuildId"/>.
+        /// </summary>
+        public ulong GuildId { get; set; }
```

Comment:
I guess it's a bit of a nitpick, but I don't see much value in comments like this since you can't easily click-through to the documentation. This is especially the case since our current architecture is designed to hide our entities from devs when they're working at the bot layer.

Reply:
I wouldn't say the intention is to HIDE the entity types, but adding a layer between them provides for an easy place to put translation and access control logic.